### DISCUSSION TITLE
Can't open any subpage on zigbee 9.5.0.5

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#12905] [Q&A] [answered]
Hi!

I upgraded from 9.4.0.4 to 9.5.0.5 but now I can't open any subpage anymore:

How can I solve this? I can't open the upgrade firmware page to downgrade also...

https://user-images.githubusercontent.com/2833940/129694768-fa7c7643-11de-4918-9954-ec2c3be79148.mp4

**Comment by barbudor** [comment]
I just tested with latest 9.5.0.6 on a NodeMCU using tasmota, tasmota-sensors, tasmota-zigbee and I can't reproduce the problem.
Same with a saved copy of 9.5.0.5 of tasmota (I don't have a saved copy of tasmota-zigbee)

**Comment by jeroenst** [comment]
I also use latest tasmota (9.5.0) on a non zigbee device (shelly) without problems.

**Reply by barbudor** [reply]
I tried again with tasmota-zigbee 9.5.0.6 having a Zigbee_RX/TX configured but no real CC hardware and can't reproduce

Can you try with latest 9.5.0.6 to be sure it's still there ?
Can you confirm that even after restarting the device, you always can access the root page ? (only sub-page timeout ?)

As I don't have Zb hardware, I can't go further. Someone with Zb hardware will need to jump in.

**Reply by jeroenst** [reply]
I want to try, but I can't open the upgrade page :-(

**Reply by jeroenst** [reply]
Only the buttons zigbee map and zigbee join do work..

**Reply by barbudor** [reply]
What device is it?
I believe serial is already used by Zb module connection so not an option
Is it connected o MQTT? You could send  'OtaUrl' and 'Upgrade' commands with a tool like TDM

By the way it could be useful to understand if only Web is down (and mqtt still working) or if everything is stuck

Another test could also if possible to disconnect the Zb module and let tasmota timeout to see if it recovers

**Reply by jeroenst** [reply]
It's a sonoff zbbride..
I'm now going to try it with an mqtt command, have search for the manual first.

**Reply by jeroenst** [reply]
Ok, I downgraded to 9.4.0 and I can open the submenus again.

**Reply by jeroenst** [reply]
Upgraded to 9.5.0 and opening submenus still works

**Reply by jeroenst** [reply]
Now I flashed 9.5.0.6 and opening submenus (except zigbee map) doesn't work anymore.

**Reply by jeroenst** [reply]
Back to 9.5.0 and everythings works again.

**Comment by arendst** [comment]
What do you use as browser. If need be execute command so128 1

**Reply by jeroenst** [reply]
Firefox and Chrome

**Reply by jeroenst** [reply]
so128 1 works and enables the buttons for submenus again.

Maybe enable so128 by default in 9.5.0.x or change the working of this feature?

**Comment by arendst** [comment]
Do you access the menus from the device or from another tool like tasmoadmin.

So128 is a new security measure to solve unwanted cross site scripting which might toggle your power unwantingly

Selecting menus from the device main menu should have worked. Requesting these menus from other sources should have resulted in your initial issue.

**Reply by jeroenst** [reply]
I do not use any tool, **i access the menus directly on the tasmota device while using chrome** (or firefox).

**Reply by jeroenst** [reply]
I also have a sonoff 4ch pro, but the problem doesn't occur with 9.5.0.6.

Maybe it's zbbridge related?

**Comment by jeroenst** [comment, marked as answer]
Solved by @arendst in https://github.com/arendst/Tasmota/pull/12900#issuecomment-902691054
